src/lib/cimb_translator/GridConf.h

```diff
@@ -22,10 +47,35 @@ namespace cimbar
 		static constexpr unsigned symbol_bits = 4;
 		static constexpr unsigned ecc_bytes = 30;
 		static constexpr unsigned ecc_block_size = 155;
-		static constexpr int image_size = 1024;
+		static constexpr unsigned image_size_x = 1024;
+		static constexpr unsigned image_size_y = image_size_x;
 
 		static constexpr unsigned cell_size = 8;
+		static constexpr unsigned cell_spacing_x = cell_size+1;
+		static constexpr unsigned cell_spacing_y = cell_size+1;
 		static constexpr unsigned cell_offset = 8;
-		static constexpr unsigned cells_per_col = 112;
+		static constexpr unsigned cells_per_col_x = 112;
+		static constexpr unsigned cells_per_col_y = cells_per_col_x;
+
+		static constexpr int fountain_chunks_per_frame = 2;
+	};
+
+	struct Conf8x8_mini
+	{
+		static constexpr unsigned color_bits = 2;
+		static constexpr unsigned symbol_bits = 4;
+		static constexpr unsigned ecc_bytes = 35;
+		static constexpr unsigned ecc_block_size = 174;
+		static constexpr unsigned image_size_x = 1285;
+		static constexpr unsigned image_size_y = 736;
```

Comment:
This is our "widescreen"/720p experiment. These params will likely change, but they do work (though not as well as I'd like for lower res cameras... which is one reason they'll change).

In short, only Conf8x8 is "stable" here -- everything else is an experiment.